unyt/tests/test_array_functions.py

```diff
@@ -1227,16 +1227,37 @@ def test_isclose_error():
         np.isclose(x, y)
 
 
-@pytest.mark.parametrize(
-    "func",
-    [
-        np.linspace,
-        np.logspace,
-        np.geomspace,
-    ],
-)
-def test_xspace(func):
-    res = func(1 * cm, 11 * cm, 10)
+@pytest.mark.parametrize("retstep", (True, False))
+def test_linspace(retstep):
```

Comment:
only 3 lines are actually shared by both variants of the test. I think in this case 2 completely separate tests are more readable than one parametrized test